lib/datasource/metadata.js

```diff
@@ -79,7 +79,12 @@ function addMetaData(dep, datasource, lookupName) {
     dep.sourceUrl = manualSourceUrls[datasource][depName];
   }
   if (dep.sourceUrl && dep.sourceUrl.includes('github.com')) {
-    dep.sourceUrl = parse(dep.sourceUrl);
+    dep.sourceUrl = parse(
```

Comment:
https://github.com/renovatebot/renovate/pull/4123#issuecomment-513137483

Our source url's are not wrong. It's just that we needed to add a massage for source urls so that they can be parsed here correctly, a source url like `https://github.com/carltongibson/django-filter/tree/master` was returning undefined after being parsed.